models/web/sources.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+
+sources:
+  - name: atomic
+    schema: atomic
+    database: snplow5
```

Comment:
These should be:
```
    schema: "{{ var('snowplow__atomic_schema', 'atomic') }}"
    database: "{{ var('snowplow__database', target.database) }}"
```

Reply:
And can you add source table descriptions? Doesn't have to be as extensive as this but it's a good idea to have something like this
https://github.com/snowplow/dbt-snowplow-web/blob/main/models/base/src_base.yml